python/cuvs/cuvs/tests/test_ivf_sq.py

```
@@ -0,0 +1,143 @@
+
+def run_ivf_sq_build_search_test(
+    n_rows=10000,
+    n_cols=10,
+    n_queries=100,
+    k=10,
+    dtype=np.float32,
+    add_data_on_build=True,
+    metric="euclidean",
+    compare=True,
+    inplace=True,
+    search_params={},
```

Ruff (0.15.13)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)